src/components/molecules/BackButton/BackButton.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
looks like it's an atom. WDYT?

Reply:
Fair enough. It's a small component. Fixed in: https://github.com/sparkletown/sparkle/pull/1492/commits/fd5e29166eac851e32064ef6881374403dd1132d